ADOPTERS.md

```diff
@@ -7,4 +7,5 @@ This really helps the project to gain momentum and credibility. It's a small con
 |--------------|---------|--------------------|
 | [Alliander](https://www.alliander.com/en/open-source/) | ospo@alliander.com | Alliander is mainly interested in the security side of the software supply chain. Alliander uses multiple tools and has evaluated even more, but currently ORT is unbeaten. ORT provides accurate and destailed scan results and allows us to define a through policy to use in evaluation. We are currently working to adopt ORT for thorough scans in addition to other tools. We're working with the community to get ORT running in GitHub Actions. |
 | [Deutsche Telekom AG](https://www.telekom.de/) | daniel.eder@magenta.at | Deutsche Telekom is in the process of adopting ORT as part of our CI/CD chain and is working on a custom ORT reporter in our [ORT fork](https://github.com/telekom/ort). |
+| [HERE Technologies](https://here.com) | opensource@here.com | HERE Technologies uses ORT to generate SBOMs for all our products, as well as to manage licensing risks. |
```

Comment:
Nits:
- Please use a trailing slash for the URL for consistency, like `https://here.com/`.
- Instead of "all *our* products" say "all *its* products" (emphasis only added for clarity).

Reply:
Can you please follow-up @bennati?

Reply:
yes, it's in my todo list